app/ancillaryFactory/src/components/NoAncillaryMessage.js

```diff
@@ -0,0 +1,16 @@
+// @flow strict
+
+import * as React from 'react';
+import { View } from 'react-native';
+import { Text } from '@kiwicom/mobile-shared';
+import { Translation } from '@kiwicom/mobile-localization';
+
+const NoAncillary = () => (
+  <View>
+    <Text>
```

Comment:
You actually don't need this text tag. The translation renders the default text tag. You just need this if you need to apply some styles.